analyzers/python/python.go

```diff
@@ -154,6 +157,12 @@ func (a *Analyzer) Analyze() (graph.Deps, error) {
 			Direct:     imports,
 			Transitive: fromImports(imports),
 		}, nil
+	case "pipenv":
+		depGraph, err := a.Pipenv.Deps()
```

Comment:
[nit] you can just return this line and avoid the nil check below

Reply:
You are absolutely right... thanks for pointing this out.